libs/core/algorithms/include/hpx/parallel/algorithms/detail/find.hpp

```diff
@@ -381,7 +381,7 @@ namespace hpx { namespace parallel { inline namespace v1 { namespace detail {
 
     template <typename ExPolicy, typename Iter1, typename Iter2, typename Token,
         typename Pred, typename Proj1, typename Proj2>
-    HPX_HOST_DEVICE HPX_FORCEINLINE Iter1 sequential_find_end(Iter1 it,
+    HPX_HOST_DEVICE HPX_FORCEINLINE void sequential_find_end(Iter1 it,
```

Comment:
Why return `void` here? Shouldn't `find_end` provide the iterator to the found element?